File: data/r/convert.py
import pandas as pd


college_qualities = pd.read_excel("college_qualities.xlsx", sheet_name=None)
college_qualities = list(college_qualities.values())[0]
college_qualities.to_csv("college_qualities.csv", encoding='utf_8_sig')

equiv_scores = pd.read_excel("equiv_scores.xlsx", sheet_name=None)
equiv_scores = list(equiv_scores.values())[0]
equiv_scores.to_csv("equiv_scores.csv", encoding='utf_8_sig')

college_admissions_simin = pd.read_excel("college_admissions_simin.xlsx", sheet_name=None)
college_admissions_simin = list(college_admissions_simin.values())[0]
college_admissions_simin = college_admissions_simin.drop(columns=['campus'])
college_admissions_simin.to_csv("college_admins.csv", encoding='utf_8_sig')

File: data/r/convert.R
library(data.table)
library(rio)

college_qualities <- fread("college_qualities.csv", encoding='UTF-8')
college_qualities <- as.data.table(college_qualities)
print(college_qualities)
export(college_qualities, "college_qualities.rds")

equiv_scores <- fread("equiv_scores.csv", encoding='UTF-8')
equiv_scores <- as.data.table(equiv_scores)
print(equiv_scores)
export(equiv_scores, "equiv_scores.rds")

college_admissions_simin <- import("college_admins.csv", encoding='UTF-8')
college_admissions_simin <- as.data.table(college_admissions_simin)
print(college_admissions_simin)
export(college_admissions_simin, "college_admins.rds")
